web/src/composebox_typeahead.js

```diff
@@ -1125,11 +1125,17 @@ export function initialize() {
         },
         sorter(items) {
             const sorted = typeahead_helper.sorter(this.query, items, (x) => x);
-            if (sorted.length > 0 && !sorted.includes(this.query)) {
```

Comment:
Please add a comment explaining why we now allow 0 originally sorted items -- it's so that we always show if the topic being typed is new, right? Also, do we really need the length check?

Reply:
We need to fix the puppeteer flake here in order to merge this PR.